### ISSUE TITLE
schedule.present state not idempotent in 2019.2.0

### ISSUE THREAD

**Issue by OrangeDog** [saltstack/salt#51824]
Master and minion 2019.2.0 (with #51265).
Every alternating application of this state reports changes.
```
highstate:
  schedule.present:
    - enabled: true
    - function: state.apply
    - job_kwargs:
        saltenv: base
    - minutes: 10
    - returner: zabbix
    - run_on_start: true
```
```
          ID: highstate
    Function: schedule.present
      Result: True
     Comment: Modifying job highstate in schedule
     Started: 12:34:57.835010
    Duration: 27.769 ms
     Changes:
              ----------
              diff:
                  ---
                  +++
                  @@ -1,10 +1,9 @@
                   enabled:True
                   function:state.apply
                   jid_include:True
                  -kwargs:{'saltenv': 'base'}
                  +kwargs:OrderedDict([('saltenv', 'base')])
                   maxrunning:1
                   minutes:10
                   name:highstate
                   returner:zabbix
                   run_on_start:True
                  -splay:None
```

**Comment by max-arnold** [comment]
Duplicate of #51467

**Comment by garethgreenaway** [comment]
Closing this out as the fix is in the 2019.2 and later branches, and will be available in the next minor releases. If the problem persists, let us know and we can reopen this issue.
